configurations/labwc/rc.xml

```diff
@@ -286,6 +288,13 @@
         <command>lxqt-leave</command>
       </action>
     </keybind>
+    <keybind key="C-A-D">
```

Comment:
Why `C-A-D`, and not `W-d` like in Openbox?
https://github.com/danakj/openbox/blob/9e8813e111cbe6c1088f6abbc771a29470f05fc2/data/rc.xml#L232-L234

Reply:
Explained below, I have to say that this was one of the first keybinds I changed to `A-D`.  But We could change it, it's acrobatic on my keyboard...

Reply:
I agree, but actually LXQt X11 uses C+A+D by default. There was always an issue about as WMs can have that too. I'm good with Meta+D too.